modules/openapi-generator/src/main/resources/go/api_doc.mustache

```diff
@@ -28,7 +28,8 @@ package main
 import (
     "context"
     "fmt"
-    "os"
+    "os"{{#vendorExtensions.x-go-import}}
+{{{vendorExtensions.x-go-import}}}{{/vendorExtensions.x-go-import}}
```

Comment:
I *think* if you just use

```
{{#vendorExtensions.x-go-import}}
{{{vendorExtensions.x-go-import}}}
{{/vendorExtensions.x-go-import}}
```

without any heading/trailing whitespace, it will still produce no output (not even blank lines) if there are no extra imports - this would make the template code nicer.